indico/modules/events/editing/settings.py

```diff
@@ -14,7 +14,13 @@
 
 
 editing_settings = EventSettingsProxy('editing', {
-    'review_conditions': OrderedDict()
+    'review_conditions': OrderedDict(),
+    'editable_types': OrderedDict([
```

Comment:
Do we want to allow changing the title? If not, I'd just map type name to enabled bool (using a normal dict)

Reply:
We probably don't want to allow changing the title. Where should I store it, if not there?

Reply:
Just a mapping in the JS i'd say...